readthedocs/builds/admin.py

```diff
@@ -19,6 +25,29 @@
 from readthedocs.search.utils import _indexing_helper
 
 
+def _pretty_config(instance):
+    """
+    Function to display pretty version of our data
```

Comment:
This should make the linter happy

```suggestion
    Function to display pretty version of our data.
```